indico/web/forms/widgets.py

```diff
@@ -294,6 +294,7 @@ def __init__(self, post_url=None, max_file_size=None, max_files=10, add_remove_l
             'parallelUploads': max_files,
             'handleFlashes': handle_flashes
         }
+        self.style = style
```

Comment:
do we? in that case i'd rather go for an enum - but i'd do that once we do end up adding 3+ styles which i find somewhat unlikely